pos_keep_draft_orders/tests/test_pos_keep_draft_orders.py

```diff
@@ -0,0 +1,133 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    POS Keep Draft Orders module for Odoo
+#    Copyright (C) 2015-Today GRAP (http://www.grap.coop)
+#    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import netsvc
+
+from openerp.tests.common import TransactionCase
+
+
+class TestPosKeepDraftOrders(TransactionCase):
+    """Tests for 'Point Of Sale - Keep Draft Orders' Module"""
+
+    def setUp(self):
+        super(TestPosKeepDraftOrders, self).setUp()
+        cr, uid = self.cr, self.uid
+
+        self.imd_obj = self.registry('ir.model.data')
+        self.ps_obj = self.registry('pos.session')
+        self.po_obj = self.registry('pos.order')
+        self.pmp_obj = self.registry('pos.make.payment')
+        self.wf_service = netsvc.LocalService('workflow')
+
+        self.pos_config_id = self.imd_obj.get_object_reference(
```

Comment:
`TransactionCase` provides shotcuts to get value from xml_id https://github.com/odoo/odoo/blob/7.0/openerp/tests/common.py#L62
```python
self.pos_config_id = self.ref('point_of_sale.pos_config_main')
```